boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+struct Imu;
+
+/** Creates Imu struct used for reading data from sensor
+ * @param update_data - pointer to function which updates data and returns true
+ * if update successful
+ * @param get_acceleration_x - pointer to function which returns x acceleration in m/s^2
```

Comment:
I would change the following comments to something like this:

``` * @param get_acceleration_x  A function that can be called to get the acceleration (m/s^2) in the x axis.```

Reply:
ok. I'll label function pointers as functions from now on